.scrutinizer.yml

```diff
@@ -1,6 +1,6 @@
 build:
     environment:
-        php: '7.3'
+        php: '7.4'
 
     tests:
         override:
```

Comment:
Scrutinizer and .scrutinizer.yml can be removed, because you don't need to run tests twice

Reply:
.scrutinizer.yml removed, but we still have Scrutinizer in validation list, how can I remove it from there?